crates/typst-library/src/compute/data.rs

```diff
@@ -130,11 +130,19 @@ pub fn csv(
     #[named]
     #[default]
     delimiter: Delimiter,
+    /// Whether to use the header row in the CSV file to generate labels the fields in the columns.
+    /// When using this option, each row in the CSV file will be represented as an dictionary, where the header field is the key and the column field is the value.
+    /// All rows will be collected into a single array. Header rows will be stripped.
+    /// This option only makes sense when a header row is present in the CSV file.
+    /// Defaults to `{false}`.
```

Comment:
The first sentence isn't grammatically correct. I think it would generally be better to use the term `key` as opposed to `label` because labels are an existing concept in Typst. Please also wrap the doc comments at 80 columns. As stated, the default will be shown automatically.